Nim/Program.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System.Diagnostics;
+using System.Globalization;
+using System.Net;
+using System.Net.Http.Headers;
+namespace ProjectNim;
+
+class nimGame
+{
+    static void Main()
+  {
+      Console.WriteLine("Welcome To Nim!");
+    bool Winning = false;
+    int CurrentMatch = 24;
+    int maxMatch = 24;
+    Random random= new Random();
+    int AIrandom;
+    
+    Console.WriteLine("|||||||||||||||||||||||| (24)");
+    Retry:
+    Console.WriteLine("You can only do 1,2 or 3 match drawn");
+    Console.WriteLine("How many matches do you want to draw?");
+    
+                 while (Winning == false)
+                 {
+                     double drawAmmount = Convert.ToDouble(Console.ReadLine());
```

Comment:
Better Convert.ToInt and use `int` because you don't want to draw `2.4` matches, right? :D